docs/walkthroughs/custom-integrating-with-bower-example.md

```diff
@@ -0,0 +1,130 @@
+# Custom Integration using `fossa-deps`
+
+With `fossa-deps.{yml, json, yaml}` file, FOSSA CLI can be integrated to support any package manager or custom and non-standard management solution, that is yet to be supported natively by FOSSA CLI. With the fossa-deps file, we can:
+
+- [Include manual dependencies](./../features/manual-dependencies.md)
+- [Include vendored dependencies](./../features/vendored-dependencies.md)
+
+## Example with Bower
+
+For an example, we will look at [Bower](https://bower.io/). Although, Bower team advises now, that users should migrate to npm, or yarn for new javascript based projects. We may have legacy or active projects, which are using bower. And we would like to include them in our reporting.
+
+In general, we can identify list of dependencies from our custom tool (in our case bower), by using configuration used or command provided. In bower case, we can inspect (1) `bower.json` or (2) output of `bower list` command. 
+
+From an example `bower.json` file, lists direct dependencies:
+
+```json
+{
+  "name": "example-project",
+  "authors": [
+    "user <user@example.com>"
+  ],
+  "description": "Example project",
+  "main": "main.js",
+  "license": "MIT",
+  "homepage": "",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
+  ],
+  "dependencies": {
+    "font-awsome": "^5.0.0",
+    "jquery": "^3.6.0"
+  }
+}
+```
+
+We can execute `bower list` command to list all of the resolved bower dependencies:
+
+```
+example-project /Users/example-user/path
+├── font-awsome#5.15.4
+└── jquery#3.6.0
+```
+
+We have two dependencies (1) jquery and (2) font-awsome.
+
+From [manual dependencies](./../features/manual-dependencies.md) documentation, we know that `bower` type dependencies are supported type as reference dependency.
+
+We can include all listed dependencies in fossa-deps.json. 
+
+```json
+{
+  "referenced-dependencies": [
+    {
+      "type": "bower",
+      "name": "jquery",
+      "version": "3.6.0"
+    },
+    {
+      "type": "bower",
+      "name": "font-awsome",
+      "version": "5.15.4"
+    }
+  ]
+}
+```
+
+Likewise, you can write short script in your language of choice to translate dependency graph produced by bower to fossa-deps file. Provided below is example python script, which parses dependency graph and produces fossa-deps file.
+
+```python
```

Comment:
Not necessary for this PR, but maybe should create a folder to hold scripts like this. I'm not sold on keeping these scripts inside the `spectrometer` repo, but we should at least keep track of everything.

If we created a directory called `experimental-scripts I think that would clearly show that we do not intend to keep the scripts updated.